### DISCUSSION TITLE
Missing assembliles?

### DISCUSSION THREAD

**Discussion by neggenbe** [coin-or/Sonnet#17] [Q&A] [answered]
Hi,
I'm trying to add the assemblies to my project so I can solve the following problem from the given example:

`     Model model = new Model();
      Variable x = new Variable();
      Variable y = new Variable();
      model.Add(2 * x + 3 * y <= 10);
      model.Objective = 3 * x + y;
      //Solver solver = new Solver(model);
      //solver.Maximise();`

The above code works but when I uncomment the Solver part, I run into the follwing error:

_System.BadImageFormatException: Could not load file or assembly 'SonnetWrapper, Version=1.4.0.0, Culture=neutral, PublicKeyToken=null'. An attempt was made to load a program with an incorrect format.
File name: 'SonnetWrapper, Version=1.4.0.0, Culture=neutral, PublicKeyToken=null'_

My intuition is that I am missing some dependencies - which ones would that be? Could you please help out?

NOTE: I see another `Ijwhost.dll` file but MSVS rejects adding it to my project.

**Comment by neggenbe** [comment, marked as answer]
Ok I found the answer...
It seems I should simply "copy" the Ijwhost.dll file to the output folder (not adding it as an assembly)...

**Comment by jhmgoossens** [comment]
Glad you found the root cause, and thanks for reporting back!

I had to google _ijwhost.dll_--which is [apparently created in the .NET Core builds](https://docs.microsoft.com/en-us/dotnet/core/porting/cpp-cli) for the C++/CLI of SonnetWrapper. 
This explains why this error only came up at the solver.Maximise() line, since that is where the SonnetWrapper code comes in to bridge to the C++ of Clp/Cbc etc. 

I've changed AppVeyor config to include the ijwhost.dll in the binary zips for net5.0 in 35d21ba.
